src/github/release/README.md

```diff
@@ -0,0 +1,48 @@
+# `garden github-release`
+
+Create a GitHub release.
```

Comment:
No `github-release` is not the same as the version script on react components. However, it decomposes and improves much of the logic contained within that script for reuse across repos. Running `github-release` creates one of [these](https://github.com/zendeskgarden/react-components/releases), for example. That's it's sole responsibility – create a GitHub release.

Reply:
Awesome, thanks for the clarification. Should that information be included in the description for this `README`? I found it helped me understand what the command does, and when it might (and might not) be used.

Reply:
Done.